src/Mod/Path/PathScripts/PathDressupRampEntry.py

```diff
@@ -63,10 +65,15 @@ def __setstate__(self, state):
         return None
 
     def onChanged(self, obj, prop):
-        if prop == "RampFeedRate":
+        if prop in ["RampFeedRate", "UseStartDepth"]:
             self.setEditorProperties(obj)
 
     def setEditorProperties(self, obj):
+        if obj.UseStartDepth:
```

Comment:
I think this will give an error with objects loaded from existing files, which don't have a UserStartDepth property, no? It might be good to add an `if hasattr(obj,"UseStartDepth"):` before...

Reply:
Good point. During 0.17 Path development I think it was pretty much agreed that there is no need to maintain compatibility of older files, but it probably makes more sense now. I'll fix tomorrow.